### PR TITLE
[FIX] Added toFixed to Delivery Modal

### PR DESCRIPTION
**Pull request by mGranTY**
# Description

Added toFixed in deliveryModal to fix the goblin image

BEFORE
image

AFTER
image

real values dont change, only the displayed ones.

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Go to the Storage in Goblin Town and try to the Delivery the Carrots.

# Checklist:

- [X] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [X] Screenshot if it includes any UI changes
- [X] I have read the contributing guidelines and agree to the T&Cs
- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by mGranTY** [comment]
Added parseFloat() so extra zeros dont show up.

if there was no parseFloat() the stone delivery would show up as 4.200
image

File: src/features/game/lib/constants.ts [hunk 1]
```diff
@@ -632,6 +632,7 @@ export const EMPTY: GameState = {
     Wood: new Decimal(50),
     Gold: new Decimal(10),
     Stone: new Decimal(10),
+    Carrot: new Decimal("334.119999999999991"),
```
Comment (spencerdezartsmith) [inline src/features/game/lib/constants.ts:635]: Can you revert this back before I merge?

Comment (mGranTY) [reply src/features/game/lib/constants.ts]: Done sir :)

Comment (mGranTY) [reply src/features/game/lib/constants.ts]: it was the only way to test the delivery in a local branch

**Review by spencerdezartsmith — APPROVED** [review]
🚀

**Pull request merged**